quinn-proto/src/connection.rs

```diff
@@ -2290,6 +2296,9 @@ pub enum ConnectionError {
     /// The peer has become unreachable.
     #[fail(display = "timed out")]
     TimedOut,
+    /// Maybe received a stateless reset.
+    #[fail(display = "maybe stateless reset (authentication failed or duplicate)")]
+    StatelessReset,
```

Comment:
This is exactly redundant to `Reset`. Note that stateless resets are not ambiguous.

Reply:
Removed it and used `Reset` instead.